### ISSUE TITLE
pytm repo is now in OWASP github space

### ISSUE THREAD

**Issue by jgadsden** [OWASP/www-project-developer-guide#290]
**Describe what change you would like** :  

pytm is now in the OWASP github space, with all forks, stars and collaborators it had before

Although not strictly necessary, as these links are redirected and work as before, it would be good to fix them up to reference OWASP now :

```
[pytmrepo]: https://github.com/izar/pytm/
[pytmexample]: https://github.com/izar/pytm/blob/master/tm.py
[pytmreleases]: https://github.com/izar/pytm/releases
```

**Context** :  
Section: `06-design/02-threat-modeling/01-pytm`

**Comment by arkid15r** [comment]
oh boy this [explains a lot](https://github.com/OWASP/Nest/issues/20), @jgadsden thanks for creating this issue

**Comment by jgadsden** [comment]
fixed in commit [2ed66a4](https://github.com/OWASP/www-project-developer-guide/commit/2ed66a4eb1fedbf8f6448d873435d4c9d3231a50)
